kolibri/core/webpack/management/commands/devserver.py

```diff
@@ -14,6 +14,10 @@
 logger = logging.getLogger(__name__)
 
 
+if os.environ['KOLIBRI_LISTEN_PORT']:
```

Comment:
We set a default value for this in `kolibri/utils/env.py`, which is `8080`, so this now makes the devserver run on `8080` by default.

Is there any reason the purposes of this PR could not be achieved just by passing the port in the command line? https://docs.djangoproject.com/en/1.11/ref/django-admin/#django-admin-runserver

Reply:
> Is there any reason the purposes of this PR could not be achieved just by passing the port in the command line?

My goal is to set up independent environments for each branch and be able to run them concurrently, alongside the `DJANGO_SETTINGS_MODULE` and `KOLIBRI_HOME` variables.

This PR makes things in line with `kolibri --help`. The command-line option is functional but much less convenient because I need to remember the weird IP syntax and remember to manually change it for each environment.

Reply:
If you have yarn > 1, you can just do `yarn run devserver 7070` to run the devserver on port 7070.

Reply:
(if you have yarn < 1, you need to do `yarn run devserver -- 7070` to pass through the argument)

Reply:
I don't quite understand...are you saying that the devserver should not respect the environment variable?

I like the idea of having the devserver by default on a different port, but it's not clear to me why that is more important than respecting environment vars.